docs/managing-e2e-tests.md

```diff
@@ -23,106 +24,140 @@ It contains a list of jobs such as:
-
-The SIG also uses another deployment tool called [kinder](https://github.com/kubernetes/kubeadm/kinder). kinder is based on kind and it's used for upgrades and version skew tests, but it does not require kubetest integration.
+The SIG also uses another deployment tool called [kinder](https://github.com/kubernetes/kubeadm/kinder).
+kinder is based on kind and it's used for upgrades and version skew tests, but it does not require kubetest integration.
 
 Kinder uses test workflow files that run sequences of tasks, such as "upgrade", "run e2e conformance tests", "run e2e kubeadm tests".
 An example of such a workflow file can be seen [here](https://github.com/kubernetes/kubeadm/blob/master/kinder/ci/workflows/upgrade-stable-master.yaml).
 
-### The testgrid configuration
-
-testgrid uses a rather large configuration file that hosts the dashboard of all the tests that can be found [here](https://k8s-testgrid.appspot.com). The initial list of menu items mostly holds Kubernetes SIGs, while the SIG Cluster Lifecycle specific dashboard can be found [here](https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-all).
+### Testgrid configuration
 
-The configuration file itself is located [here](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml). The file is broken into multiple sections. If you wish to find the SIG Cluster Lifecycle and kubeadm tests you can search for `sig-cluster-lifecycle`. For instance, `sig-cluster-lifecycle-all` has all the relevant kubeadm tests.
+Testgrid contains elements like dashboard groups, dashboards and tabs.
 
-A dashboard is defined like so:
-```
-- name: sig-cluster-lifecycle-kubeadm
-  dashboard_tab:
-# kubeadm-kind tests
-  - name: kubeadm-kind-...
-    test_group_name: ci-kubernetes-e2e-kubeadm-kind-...
-    ...
-  - name: kubeadm-kind-...
-    test_group_name: ci-kubernetes-e2e-kubeadm-kind-...
-```
+As an overview:
+- SIG Cluster Lifecycle dashboards reside in a dashboard group that can be found [here](https://k8s-testgrid.appspot.com).
+- Inside this dashboard group there is a [dashboard for kubeadm](https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-kubeadm).
+- Inside this dashboard there are individual tabs such as `kubeadm-kind-master` (which is the tab name for the
+job `ci-kubernetes-e2e-kubeadm-kind-master`).
 
-What the above does is defining a dashboard called `sig-cluster-lifecycle-all` and placing inside a couple of tests defined by `name` and linking to a `test_group_name`. Some items may also have a `description`.
+In earlier versions of testgrid, all the different test jobs of different SIGs were contained in single
```

Comment:
The history is informative, but probably it is better to keep things simple and describe only the current state. wdyt?

Reply:
removing the historic info